src/common/listActions/ListActions.tsx

```diff
@@ -39,20 +43,20 @@ const ListActions = <T extends object | string | number | boolean | bigint | sym
   const selectedAction = listActions.find((action) => action.id === selectedActionId);
 
   return (
-    <div className={styles.container}>
+    <div className={classNames(styles.container, className)}>
       <Select
         onChange={(e) => setSelectedActionId(listActions.find((action) => action.id === e.target.value)?.id)}
         value={selectedActionId ?? ''}
         options={options}
         className={styles.select}
       />
-      {selectedAction?.renderComponent?.()}
+      {selectedAction?.renderComponent?.(() => setSelectedActionId(undefined))}
```

Comment:
What happens here?

Reply:
I pass a `resetSelection` callback to the `renderComponent` method.
But that's right, it's confusing to implement it inline so I extracted it to a named function